snorkel/classification/training/trainer.py

```diff
@@ -12,7 +12,12 @@
     ClassifierConfig,
     SnorkelClassifier,
 )
-from snorkel.classification.training import (
+from snorkel.types import Config
+from snorkel.utils.config_utils import merge_config
+from snorkel.utils.lr_schedulers import LRSchedulerConfig
+from snorkel.utils.optimizers import OptimizerConfig
```

Comment:
Are these no user facing? (since they are three levels deep).

Reply:
correct, they are not